Gemfile

```diff
@@ -1,16 +1,26 @@
 source 'https://rubygems.org'
 
-gemspec
-
 group :development do
-  gem 'vagrant',      git: 'https://github.com/mitchellh/vagrant.git'
-  gem 'rake'
+  gem 'vagrant',
+      :git => 'git://github.com/mitchellh/vagrant.git',
+      :ref => 'v1.8.1'
+
+  # test dependencies
+  gem 'minitest'
+  gem 'mocha'
+
   gem 'cucumber', '~> 2.1'
   gem 'aruba', '~> 0.13'
   gem 'komenda', '~> 0.1.6'
   gem 'launchy'
   gem 'gem-compare'
   gem 'mechanize'
+
+  # build tool
+  gem 'rake', '10.4.2'
```

Comment:
@hferentschik lets go for the latest i.e. 1.8.4. I have not used 1.8.4 but there were some blocker issues in 1.8.1 and I am expecting some of them might have been fixed in 1.8.4.